packages/button/button.scss

```diff
@@ -9,21 +11,32 @@ $button-focus-ring-color: $himmel;
 
 $button-text-color: $svart;
 $button-text-color--primary: $snøhvit;
-$button-font-size: $font-size-2;
-$button-height: 2.5rem;
-$button-width: $button-height;
 $button-border-width: rem(1px);
 $animation-timing: 100ms cubic-bezier(0.6, 0.2, 0.35, 1);
 $hover-elevation-distance: -0.25rem;
 
+$button-height--large: 3.5rem;
+$button-height--small: 2.5rem;
+
+@mixin deviceScale() {
```

Comment:
Does this need to be a mixin? Feels like it would be clearer if it was just  part of the class style..